### ISSUE TITLE
low text color contrast on Dashboard

### ISSUE THREAD

**Issue by diminutivesloop** [swdotcom/swdc-vscode#337]
**Describe the bug**
```paste below
The Dashboard view is not readable due to low text color contrast.
```
**ISSUE TYPE**
- Bug Report

**Screenshots**
<img width="913" alt="image" src="https://user-images.githubusercontent.com/667716/137353057-08298e09-0768-4088-9303-bd4cba830459.png">

**Desktop (please complete the following information):**
 - OS: Mac OS
 - Plugin version: 2.5.27

**Comment by mmclead** [comment]
Hi @diminutivesloop.  Thanks for submitting the issue.  We'll take a look today.

What theme are you using on VS Code?   

This screenshot I just took is using Dark+ (default dark)
![Screen Shot 2021-10-14 at 9 33 21 AM](https://user-images.githubusercontent.com/898851/137359434-8766462f-fe29-4570-ae0a-f9a17733e133.png)

**Comment by diminutivesloop** [comment]
I'm using [Atom One Light](https://marketplace.visualstudio.com/items?itemName=akamud.vscode-theme-onelight) but I cycled through the other themes and it didn't have any effect.

**Comment by xavluiz** [comment]
Hi @diminutivesloop, we've launched v2.6.1 to fix this issue, as well with other optimizations. Please let us know if there are any other issues, thank you.
